apps/webapp/src/app/meeting-add/slots-editor.tsx

```diff
@@ -73,23 +73,54 @@ export const SlotsEditor: FC<AddSlotsComponentProps> = (props) => {
               />
             )}
           />
-          <TextField
-            style={{ marginRight: '15px', width: '150px' }}
+          <DesktopTimePicker
             label="Od"
-            value={slot.timeFrom ?? ''}
-            onChange={(event) => {
-              handleTimeFromChange(slot.id, event.target.value);
+            inputFormat="HH:mm"
+            mask="__:__"
+            disableOpenPicker={true}
+            onChange={(value) => {
```

Comment:
pewnie bym wolał oneliner `onChange={(value) => handleTimeFromChange(slot.id, value ? dayjs.formatByString(value, 'HH:mm') : '')}` ale to może być moje zboczenie ;)